scripts/ci/VM-container-commands.sh

```diff
@@ -186,3 +186,9 @@ cmd_control_reboot() {
 	do_test_cmd_noutput "/usr/sbin/control reboot" "Abort"
 }
 
+cmd_control_get_all_installed_versions(){
+	COMMAND="/usr/sbin/control list_guestos | grep version\:"
+	OUTPUT="$(ssh ${SSH_OPTS} "$COMMAND")" || true
+	dbg "Command returned $OUTPUT, code: $?"
+	echo "$OUTPUT" | awk '{print $2}' > /tmp/installed_guestos_versions
```

Comment:
Not sure what exactly you mean by this? i adapted it to be similar to, e.g., line 63/85?

Reply:
The tmp file is not necessary here,
instead plz remove the debugging output and instead echo "$OUTPUT" | awk '{print $2}' to stdout
s.t. it can be captured via MYVAR="$(cmd_control_get_guestos_versions)"

Reply:
Fixed this code part.